docs/usage/applications.rst

```diff
@@ -25,6 +25,7 @@ and Route Handlers should be registered on it.
 
     * :ref:`Routing - Registering Routes <usage/routing/overview:Registering Routes>`
 
+.. _startup-and-shutdown:
```

Comment:
I thought that with [`autosectionlabel`](https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#module-sphinx.ext.autosectionlabel) we didn't need to do things like this

Reply:
Updated to use section reference.

Only did a quick research on rst yesterday and didnt see that as an option, but it's cleaner.

Reply:
That didnt work out. Does it need a full path? Like `<usage/application:Startup and Shutdown>`?

Reply:
You just need to get rid of the `:ref:` role, then it should work.